src/Components/HomePage.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, useState } from 'react'
+import {
+    Box,
+    List,
+    Snackbar
+} from '@material-ui/core';
+
+import MuiAlert from '@material-ui/core/Alert';
+
+import Task from './Task';
+import TaskForm, { ITask } from './TaskForm';
+
+const HomePage: FC = () => {
+
+    const [tasks, setTasks] = useState([{
+        title: 'Tarefa 1',
+        taskId: Math.random()
+    }])
+
+    const [modalActionTask, setModalActionTask] = useState({
+        status: false,
+        message: ''
+    })
+
+    const taskAddedHandler = (values: ITask ) => {
+        
+        setTasks((prevState) => [
+            {
+                title: values.title,
+                taskId: Math.random()
+            },
+            ...prevState
+        ])
+        setModalActionTask({
+            status: true,
+            message: 'Tarefa adicionada com sucesso.'
+        })
+    }
+
+    const removeTaskHandler = (taskId: number) => {
+
+        const newTasks = tasks.filter((task) => task.taskId !== taskId)
+
+        setTasks(newTasks)
+
+        setModalActionTask({
+            status: true,
+            message: 'Tarefa removida com sucesso.'
+        })
+    }
+
+    const handleClose = (_: any, reason: string) => {
+        if (reason === 'clickaway') {
+            return;
+        }
```

Comment:
I suggest applying clean code here:
```suggestion
        const isClickingAway = reason === 'clickaway';
        if (isClickingAway) {
            return;
        }
```

Reply:
Resolved on 6c5a7c7be7b5123a4deddd43c2db7289ac2338be